CONTRIBUTING.md

```diff
@@ -1,81 +1,60 @@
-# Contributing to the **Spice** repository
+# Contribute to Spice Instant Answers
 
+There are several options for contributing Spice Instant Answers, explained below.
 
-At DuckDuckGo, we truly appreciate our community members taking the time to
-contribute to our open-source repositories. In an effort to ensure
-contributions are easy for you to make and for us to manage, we have written
-some guidelines that we ask our contributors to follow so that we can handle
-pull requests in a timely manner with as little friction as possible.
+You can always find the [Instant Answer Documentation here](https://duck.co/duckduckhack/ddh-intro). Also, if you have any questions at any point, feel free to ask on one of our community channels:
 
+- [DuckDuckHack mailing list](https://duck.co/redir/?u=https%3A%2F%2Fwww.listbox.com%2Fsubscribe%2F%3Flist_id%3D197814)
+- [Gitter chat](https://duck.co/redir/?u=https%3A%2F%2Fgitter.im%2Fduckduckgo%2FIdeas)
+- Do not hesitate to email us directly at [open@duckduckgo.com](mailto:open@duckduckgo.com).
 
-## Getting Started
+## New? Make Your First Contribution Today
 
-Before you can do anything, you first need a [GitHub account](https://github.com/signup/free). This is required because we use GitHub to handle all incoming *Pull Requests* (code modifications) and *Issues* (bug reports) which cannot be made without a GitHub account.
+If this is your first time contributing to [DuckDuckHack](http://www.duckduckhack.com), you have two great ways to quickly make your first commit. Although this is the Spice repository, building a [Goodie](https://github.com/duckduckgo/zeroclickinfo-goodies) is a great introduction to creating Instant Answers.
 
+**1. Make a [Goodie Cheat Sheet](https://duck.co/duckduckhack/goodie_cheat_sheets)**
 
-## Submitting a **Bug** or **Suggestion**
+Cheat sheets are a super-easy way to contribute to the live DuckDuckGo AnswerBar very quickly, by editing a single file. Cheat sheets can be about anything, from Emacs and Vim to Game of Thrones house names or wine pairings.
```

Comment:
- Maybe instead of "to the live DuckDuckGo AnswerBar" we can use "to DuckDuckGo"? Sounds simpler to me, and doesn't add unnecessary jargon like "AnswerBar".

- Maybe it would be better of "Emacs", "Vim", and "Game of Thrones" linked to https://duckduckgo.com/?q=emacs+cheat+sheet&ia=cheatsheet, https://duckduckgo.com/?q=game+of+thrones+cheat+sheet, and https://duckduckgo.com/?q=vim+cheat+sheet&ia=cheatsheet.